tests/manifests/dvMissingSource.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: test-dv
 spec:
   source:
+  sourceRef:
```

Comment:
why is this needed? it not being there is the same as leaving it blank no?

Reply:
No real reason. Just because it was there already for `source`.
So may we remove this yaml together with `test_id:1762` and keep only `test_id:1861`. They are simillar except the blank pvc.
```
$ diff tests/manifests/dvMissingSource.yaml tests/manifests/dvMissingSource2.yaml
6,7d5
<   source:
<   sourceRef:
```